content/src/service/deployments/DeploymentManager.ts

```diff
@@ -86,6 +86,13 @@ export class DeploymentManager {
     }
   }
 
+  async getActiveDeploymentByContentHash(
+    deploymentsRepository: DeploymentsRepository,
+    hash: string
+  ): Promise<EntityByHash[]> {
```

Comment:
I think it's kind of weird to return a list of objects that can be null, specially because I don't think it can happen. Maybe we could return `EntityId[]`?

Reply:
Yes, now that we're using map it should be entityId